polaris-react/src/components/Box/Box.tsx

```diff
@@ -258,7 +246,13 @@ export const Box = forwardRef<HTMLElement, BoxProps>(
       '--pc-box-min-height': minHeight,
       '--pc-box-min-width': minWidth,
       '--pc-box-max-width': maxWidth,
-      '--pc-box-outline': outline ? `var(--p-border-${outline})` : undefined,
+      '--pc-box-outline-color': outlineColor
+        ? `var(--p-color-${outlineColor})`
+        : undefined,
+      '--pc-box-outline-style': outlineStyleValue,
+      '--pc-box-outline-width': borderWidth
+        ? `var(--p-outline-width-${borderWidth})`
+        : undefined,
```

Comment:
Should this be using `outlineWidth` instead of `borderWidth`?
I also updated the token from `--p-outline-width--` to `--p-border-width` since we don't currently have `outline` tokens.

```suggestion
      '--pc-box-outline-width': outlineWidth
        ? `var(--p-border-width-${outlineWidth})`
        : undefined,
```